api/src/pcapi/routes/backoffice/templates/offer_price_limitation_rules/list_rules.html

```diff
@@ -0,0 +1,67 @@
+{% from "components/forms.html" import build_filters_form with context %}
+{% from "components/turbo/lazy_modal.html" import build_lazy_modal with context %}
+{% from "components/turbo/spinner.html" import build_loading_spinner with context %}
+{% extends "layouts/connected.html" %}
+{% block page %}
+  <div class="pt-3 px-5">
+    <h2 class="fw-light">Règles de modification de prix d'offres</h2>
+    <div class="mt-4">
+      {{ build_filters_form(form, dst) }}
+      <button class="btn btn-outline-primary lead fw-bold mt-2"
```

Comment:
Détail : Je trouve le bouton trop près de la liste
```suggestion
      <button class="btn btn-outline-primary lead fw-bold mt-2 mb-3"
```